src/styles/submit.styles.js

```diff
@@ -0,0 +1,176 @@
+import styled from "styled-components";
+import { GREEN, P, H3, CREAM, BLUE, YELLOW, BLACK, fonts } from "./styles";
+import { min } from "./breakpoints";
+import Cursor from "../images/cursor-pointer.svg";
+import { ArrowUpCircle } from "react-feather";
+
+export const HeaderImage = styled.img`
+  width: 100%;
+`;
+
+export const ContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.desktop} {
+    flex-direction: row;
+  }
+`;
+
+export const InformationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.desktop} {
+    width: 33%;
+  }
+`;
+
+export const FormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.tablet} {
+    margin-left: 12.5%;
+    width: 75%;
+  }
+  ${min.desktop} {
+    margin-left: 8.33%;
+    width: 50%;
+  }
+`;
+
+export const Input = styled.input`
+  type: text;
```

Comment:
when I checkout this branch on my computer I get a warning that this isn't a valid css property; you can delete it! you could set the type attribute in the html but inputs are text by default so removing it altogether is fine!